frontend/scripts/gameLogicInterface.js

```diff
@@ -191,11 +191,27 @@ export default function createGame(initialGameState = emptyGameState) {
 		moveLeft: function() {
 
 		},
+          
+
+        moveDown: function() {
+			let { x, y } = this.gameState.activeTetromino.position;
+			
+			this.gameState.activeTetromino.position = { x: x, y: y - 1 };
+		if (!this.isStateValid(this.gameState.activeTetromino)) {
+			this.gameState.activeTetromino.position = { x: x, y: y };
+			return false;
+		
+		
+		
+		}
+		return true;
+		},          
+
 
 		/**
 		 * Move the current tetromino right 1 tile
 		 */
-		moveRight: function() {
+		 moveRight: function() {

```

Comment:
You've added a space here - let's keep changes to where we actually edit the code, otherwise you might end up with conflicts when merging this with other people's code!